package.json

```diff
@@ -31,36 +31,36 @@
   },
   "dependencies": {
     "@types/cors": "^2.8.5",
-    "@types/express": "^4.11.1",
+    "@types/express": "^4.17.0",
     "@types/jsonwebtoken": "^8.3.2",
-    "@types/lodash": "^4.14.133",
-    "cors": "^2.8.4",
+    "@types/lodash": "^4.14.135",
+    "cors": "^2.8.5",
     "express": "^4.17.1",
-    "jsonwebtoken": "^8.3.2",
-    "lodash": "^4.6.1"
+    "jsonwebtoken": "^8.5.1",
+    "lodash": "^4.17.11"
   },
   "devDependencies": {
     "@types/chai": "^4.1.7",
     "@types/chai-as-promised": "^7.1.0",
     "@types/mocha": "^5.2.7",
     "@types/mock-require": "^2.0.0",
-    "@types/nock": "^10.0.2",
-    "@types/node": "^8.10.49",
-    "@types/sinon": "^7.0.12",
+    "@types/nock": "^10.0.3",
+    "@types/node": "^12.0.10",
```

Comment:
This should still be 8, as we're not even supporting 12 now.

Reply:
Nice catch, changed to `^8.10.50`.